packages/strapi-parts/src/EmptyStateLayout/EmptyStateLayout.js

```diff
@@ -18,7 +19,14 @@ const EmptyStateImageWrapper = styled(Box)`
 
 export const EmptyStateLayout = ({ icon, content, action }) => {
   return (
-    <EmptyStateWrapper background="neutral0" hasRadius shadow="tableShadow" paddingTop={10} paddingBottom={10}>
+    <EmptyStateWrapper
+      padding={11}
+      background="neutral0"
+      hasRadius
+      shadow="tableShadow"
+      paddingTop={10}
+      paddingBottom={10}
```

Comment:
`padding={10}` overrides `paddingTop` and `paddingBottom`, you can remove these two lines